extensions/transmission/src/utils/darkMode.ts

```diff
@@ -0,0 +1,20 @@
+import { environment } from "@raycast/api";
```

Comment:
I couldn't find a Raycast-native way to obtain the current macOS theme. I need this to properly render the graph depending by the dark or light background.

Reply:
That's exactly why we have added the feature 😉